ui/package.json

```diff
@@ -95,7 +95,7 @@
     "shebang-loader": "^0.0.1",
     "snake-case": "^3.0.3",
     "subscriptions-transport-ws": "^0.11.0",
-    "taskcluster-client-web": "^44.20.4",
+    "taskcluster-client-web": "^44.23.3",
```

Comment:
I wonder if there is a reason we are not doing this on release?

Reply:
I suspect it's because it would require that the package already be on npm and available for download.  Maybe there's a way to refer to this by path?  I vaguely recall trying that and getting confused by webpack and neutrino and babel.

Reply:
ah, right.. publishing step is missing. I'll check if we can reference client differently here without breaking docker image builds and confusing webpack